src/RayTracer/Public/js/lib/Detector.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Can we actually look into importing these from their website rather than having to save them? 
If they're not reliably hosted we can keep them

Reply:
Okay I've had a bit of a look and I only see these online in a repo so I guess we're okay keeping them